modules/gapi/src/api/gcomputation.cpp

```diff
@@ -172,50 +226,25 @@ void cv::GComputation::apply(const std::vector<cv::Mat> &ins,
 }
 
 // NB: This overload is called from python code
-cv::GRunArgs cv::GComputation::apply(GRunArgs &&ins, GCompileArgs &&args)
+cv::GRunArgs cv::GComputation::apply(const cv::detail::ExtractArgsCallback &callback,
+                                           GCompileArgs                   &&args)
 {
-    recompile(descr_of(ins), std::move(args));
+    // NB: Used by python bridge
```

Comment:
It's a class method, don't understand how to move to `misc/python`. It would be cool to keep all python stuff separately. 
Considering to make a wrapper and put it in `python/misc` in next PR's.